src/main.cpp

```diff
@@ -55,4 +57,67 @@ void loop(){
     comms_gps(can_position->x, can_position->y, can_position->z);
 
     // Liberar memoria alocada
+}
+*/
+
+
+/*
+arriba derecha 160
+abajo derecha 30
+
+abajo izquierda 160
+arriba izquierda 25
+
+*/
+int angle = 90;
+int serie;
```

Comment:
Always initialize integers
```suggestion
int serie = 0;
```